### ISSUE TITLE
Dungeon Manjiro (MSX Magazine) (1988) hung in Turbo R mode

### ISSUE THREAD

**Issue by Dalekamistoso** [openMSX/openMSX#1383]
It hangs with a DL HALT or shows a white screen after starting the game, the game works properly on real Turbo R machines and RuMSX emulator, the game also works on openMSX  with MSX2/MSX2+ configurations, so this bug affects only Turbo R modes (tested with both models).

Tested in openMSX versions 0.17 stable  latest openMSX builds (the very same behaviour is showed by Bluemsx):

![bug-002](https://user-images.githubusercontent.com/67012746/140613086-c33ca2dd-3aba-4cd6-844d-f69555aa5f1b.png)
![bug-001](https://user-images.githubusercontent.com/67012746/140613087-00809ba1-27cc-4b8a-aed8-49197e03f18d.png)

Thanks in advance

**Comment by MBilderbeek** [comment]
I tried it on a GT in openMSX, and it hung while it was loading after starting and a screen some Japanese text and at the bottom "NOW - LOADING". The PAUSE LED went on and then nothing happened anymore. No di; halt, though.
Then I tried on an FS-A1ST, and it hung up with a di;halt after a lot of loading in the same screen indeed. 
Note that I never saw a white screen!
I used a disk image found somewhere here https://download.file-hunter.com/Games/MSX2/DSK/

I find it hard to test on my real turboR, as its disk drive is broken. @Dalekamistoso  How did you test it on real hardware? With a floppy disk?

**Comment by Dalekamistoso** [comment]
Besides from floppy you can use Nextor´s emufile as MegaFlashrom does (also it works with Flashjacks own loader)

**Comment by MBilderbeek** [comment]
@Dalekamistoso Yes, but these loaders/emulations may also cause issues. So the best test is to test this with real floppy on a real turboR (i.e. the same situation as which is emulated). Did you try that?

**Comment by MBilderbeek** [comment]
I tested on my real GT with the Nowind interface, but it works fine there... so probably it works on real MSX, but the timing will be different with a real floppy disk (which is being emulated in openMSX), so that may explain it as well.

**Comment by MBilderbeek** [comment]
With totally different timing (running from emulated IDE disk) it also hangs up in openMSX. So yeah, it's likely a bug in openMSX turboR's timing. @m9710797 care to investigate?
